crates/cheatcodes/src/env.rs

```diff
@@ -4,7 +4,11 @@ use crate::{string, Cheatcode, Cheatcodes, Error, Result, Vm::*};
 use alloy_dyn_abi::DynSolType;
 use alloy_primitives::Bytes;
 use alloy_sol_types::SolValue;
-use std::env;
+use once_cell::sync::OnceCell;
+use std::{env, mem::discriminant};
+
+/// Stores the forge execution context for the duration of the program.
+static FORGE_CONTEXT: OnceCell<ForgeContext> = OnceCell::new();
```

Comment:
use `std::sync::OnceLock`

Reply:
updated in https://github.com/foundry-rs/foundry/pull/7377/commits/d3d661cca7cc34075d31a74023b8e34c248c0c72